File: docs/modules/boozsoft-server/src/main/kotlin/org/boozsoft/repo/stock/StockCcodesRepository.java
package org.boozsoft.repo.stock;

import org.boozsoft.domain.entity.stock.StockCcodes;
import org.springframework.data.repository.reactive.ReactiveCrudRepository;
import reactor.core.publisher.Flux;

public interface StockCcodesRepository extends ReactiveCrudRepository<StockCcodes, String> {

    Flux<StockCcodes> findByUniqueIdOrderById(String uniqueId);
    Flux<StockCcodes> deleteAllByUniqueId(String uniqueId);

}

File: packages/boozsoft-platform-system/src/main/kotlin/org/boozsoft/repo/stock/StockCcodesRepository.kt
package org.boozsoft.repo.stock

import org.boozsoft.domain.entity.stock.StockCcodes
import org.springframework.data.repository.reactive.ReactiveCrudRepository
import reactor.core.publisher.Flux

interface StockCcodesRepository : ReactiveCrudRepository<StockCcodes, String> {
    fun findByUniqueIdOrderById(uniqueId: String): Flux<StockCcodes>
    fun deleteAllByUniqueId(uniqueId: String): Flux<StockCcodes>
}